deploy/helm/sumologic/templates/setup/setup-job.yaml

```diff
@@ -14,18 +14,14 @@ metadata:
 spec:
   template:
     spec:
-      restartPolicy: Never
+      restartPolicy: OnFailure
```

Comment:
Will this cause it to never eventually die?

Reply:
yeah, my thinking was we shouldn't proceed if the setup fails, since it will be missing collectors/sources/namespace/secret. By restarting on failure, it gives them the chance to modify the config map to fix the issue -OR- cancel the helm install and skip `setupEnabled` if they want to manage it manually.